### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by shakedhagag**
Added the landing page (login) with the logo + the bootstrap for it

image

### CODE REVIEW

File: static/css/landingStyle.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
+
```
Comment (Yarboa) [inline static/css/landingStyle.css:1]: We use boostrap instead of css,
boostrap is a library that wraps css

**Review by Yarboa — COMMENTED** [review]
I will wait other reviewers to share their thoughts

File: static/css/landingStyle.css (hunk 1 above)
Comment (shakedhagag) [reply static/css/landingStyle.css:1]: I did use bootstrap, I just ran over it with css when needed,  in order to make the site more responsive and nice
see index.html the <link> tag in the head section 'imports' the bootstrap in my understanding

Comment (Yarboa) [reply static/css/landingStyle.css:1]: Correct but you should use boostrap please refer this 
https://www.w3schools.com/bootstrap4/bootstrap_get_started.asp

You could play with and have immediate response for learning

Comment (shakedhagag) [reply static/css/landingStyle.css:1]: Yes, I understand, I will use bootstrap when needed. bootstrap + css is much more powerful imo
I also checked previous projects and saw that they used bootstrap and css

**Comment by Yarboa** [comment]
No other reviewers commented on this PR

**Pull request merged**